app/governance/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
we have a component called toggle Group, that might be easier to use instead of rebuilding